.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+AllCops:
+  Include:
+    - 'lib/statsample-glm/glm/formula.rb'
+    - 'lib/statsample-glm/glm/regression.rb'
```

Comment:
You can play with include/exclude [settings](http://rubocop.readthedocs.io/en/latest/configuration/#includingexcluding-files) in `.rubocop.yml`. As far as I can recall, it is not very straightforward, but manageable to include and exclude virtually any sets of files.
Let me know if it would became hard, I'll try to experiment myself.